### DISCUSSION TITLE
Issue Report

### DISCUSSION THREAD

**Discussion by changcheng967** [changcheng967/Kata_speed#10] [Announcements]
Create issue here and we'll make it live

**Comment by changcheng967** [comment]
?

**Comment by DouletMedia** [comment]
https://github.com/Doulet-Media/Kata_speed

**Comment by changcheng967** [comment]
You created this

**Comment by changcheng967** [comment]
brilliant

**Comment by DouletMedia** [comment]
The file is likely designed for unix and not for Windows so some of the files are causing error and when edit it will be worse. Need to find a way

**Reply by changcheng967** [reply]
#107
